src/main.rs

```diff
@@ -466,7 +466,7 @@ async fn main() -> Result<()> {
     }
 
     let (mut has_templates, mut template_tasks) =
-        expand_template_tasks(&chompfile, &mut extension_env)?;
+        expand_template_tasks(&chompfile, &mut extension_env, cwd.to_string_lossy().to_string().as_str())?;
```

Comment:
Just a direct reference should work:

```suggestion
        expand_template_tasks(&chompfile, &mut extension_env, &cwd.to_string_lossy().to_string())?;
```

You may need to make it a variable though:

```suggestion
        let cwd_str = cwd.to_string_lossy().to_string();
        expand_template_tasks(&chompfile, &mut extension_env, &cwd_str)?;
```